app/models/mixins/authentication_mixin.rb

```diff
@@ -12,6 +12,33 @@ def self.authentication_check_schedule
       assocs = zone.respond_to?(assoc) ? zone.send(assoc) : []
       assocs.each { |a| a.authentication_check_types_queue(:attempt => 1) }
     end
+
+    def self.queue_authentication_check(args, user_id, zone)
```

Comment:
Hi @mkanoor - I believe that most calls block the UI, so I would want to stay consistent with what's already done. I may be wrong though - are there examples of tasks using notifications for filling out a form? cc: @dclarizio @AparnaKarve 

I will look into `put_authentication_check`, thanks!